dashboard/utils.py

```diff
@@ -278,15 +278,12 @@ def get_all_final_grades(self):
         Returns:
             dict: dictionary of course_ids: floats representing final grades for a course
```

Comment:
It looks like the value in the dict is actually a `FinalGrade` object. Can you change the description here?